app/views/layouts/provider.html.slim

```diff
@@ -3,7 +3,7 @@ html[lang="en"]
   head
     meta[charset="utf-8"]
     title
-      = content_for?(:title) ? yield(:title) : default_title
```

Comment:
I think I wouldn't do this. The reason it goes wrong with title is because it's inserted outside of the wrapper which breaks the rendering. So we could do something like:

```          main#content.self_clear
            - if controller.sublayout
              = render partial: "/layouts/#{controller.sublayout.to_s}"
            - else
              - if content_for :title
                h1 = content_for :title
              = yield
```

and in the sublayouts where needed: 

```<div id="tab-content">
  <% if content_for :title %>
    <h1><%= content_for :title %></h1>
  <% end %>
  <%= yield  %>
</div>```

This way we keep using title and can dry up the page titles.